sdks/go/pkg/beam/pipeline.go

```diff
@@ -60,7 +60,8 @@ func (s Scope) String() string {
 // Pipelines can safely be executed concurrently.
 type Pipeline struct {
 	// real is the deferred execution Graph as it is being constructed.
-	real *graph.Graph
+	real               *graph.Graph
+	ExpandedTransforms map[string]*ExternalTransform
```

Comment:
As discussed, we don't want to have to hang the expanded transforms on the pipeline here.

Reply:
Removed from pipeline.